lib/log_test.go

```diff
@@ -27,23 +27,28 @@ func TestHistoryRequestsLog(t *testing.T) {
 
 	firstRef := refs[0].String()
 
-	items := make([]dsref.VersionInfo, len(refs))
+	items := make([]DatasetLogItem, len(refs))
 	for i, r := range refs {
-		items[i] = reporef.ConvertToVersionInfo(&r)
-		items[i].MetaTitle = ""
-		items[i].BodyRows = 0
-		items[i].NumErrors = 0
-		items[i].BodyFormat = ""
+		ds := r.Dataset
+		items[i].VersionInfo = reporef.ConvertToVersionInfo(&r)
+		items[i].VersionInfo.MetaTitle = ""
```

Comment:
Can get rid of these `VersionInfo` accesses here (after the first one which assigns the whole struct). Just do `items[i].MetaTitle = ""` etc.